// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dart-json tests matches report snapshot 1`] = `
Object {
  "annotations": Array [
    Object {
      "annotation_level": "failure",
      "end_line": 13,
      "message": "Expected: <2>
  Actual: <1>


package:test_api          expect
test\\\\main_test.dart 13:9  main.<fn>.<fn>.<fn>
",
      "path": "test/main_test.dart",
      "start_line": 13,
      "title": "[test\\\\main_test.dart] Test 1 Test 1.1 Failing test",
    },
    Object {
      "annotation_level": "failure",
      "end_line": 2,
      "message": "Exception: Some error

package:darttest/main.dart 2:3  throwError
test\\\\main_test.dart 17:9        main.<fn>.<fn>.<fn>
",
      "path": "lib/main.dart",
      "start_line": 2,
      "title": "[test\\\\main_test.dart] Test 1 Test 1.1 Exception in target unit",
    },
    Object {
      "annotation_level": "failure",
      "end_line": 24,
      "message": "Exception: Some error

test\\\\main_test.dart 24:7  main.<fn>.<fn>
",
      "path": "test/main_test.dart",
      "start_line": 24,
      "title": "[test\\\\main_test.dart] Test 2 Exception in test",
    },
    Object {
      "annotation_level": "failure",
      "end_line": 5,
      "message": "TimeoutException after 0:00:00.000001: Test timed out after 0 seconds.

dart:isolate  _RawReceivePortImpl._handleMessage
",
      "path": "test/second_test.dart",
      "start_line": 5,
      "title": "[test\\\\second_test.dart] Timeout test",
    },
  ],
  "testRuns": Array [
    TestRunResult {
      "path": "fixtures/dart-json.json",
      "suites": Array [
        TestSuiteResult {
          "groups": Array [
            TestGroupResult {
              "name": "Test 1",
              "tests": Array [
                TestCaseResult {
                  "name": "Test 1 Passing test",
                  "result": "success",
                  "time": 36,
                },
              ],
            },
            TestGroupResult {
              "name": "Test 1 Test 1.1",
              "tests": Array [
                TestCaseResult {
                  "name": "Test 1 Test 1.1 Failing test",
                  "result": "failed",
                  "time": 20,
                },
                TestCaseResult {
                  "name": "Test 1 Test 1.1 Exception in target unit",
                  "result": "failed",
                  "time": 6,
                },
              ],
            },
            TestGroupResult {
              "name": "Test 2",
              "tests": Array [
                TestCaseResult {
                  "name": "Test 2 Exception in test",
                  "result": "failed",
                  "time": 12,
                },
              ],
            },
          ],
          "name": "test\\\\main_test.dart",
          "totalTime": undefined,
        },
        TestSuiteResult {
          "groups": Array [
            TestGroupResult {
              "name": null,
              "tests": Array [
                TestCaseResult {
                  "name": "Timeout test",
                  "result": "failed",
                  "time": 37,
                },
                TestCaseResult {
                  "name": "Skipped test",
                  "result": "skipped",
                  "time": 14,
                },
              ],
            },
          ],
          "name": "test\\\\second_test.dart",
          "totalTime": undefined,
        },
      ],
      "totalTime": 3760,
    },
  ],
}
`;
